services/static-webserver/Dockerfile

```diff
@@ -10,21 +10,21 @@ ENV SC_USER_ID=8004 \
 
 RUN adduser -D -u ${SC_USER_ID} -s /bin/sh -h /home/${SC_USER_NAME} ${SC_USER_NAME}
 
-# front-end client last (image name is the path to the Dockerfile)
-COPY --from=client/tools/qooxdoo-kit/builder:latest --chown=${SC_USER_NAME}:${SC_USER_NAME} \
-  /project/build-output "/static-content"
-
 # changing ownership of static-web-server files
-RUN chown -R ${SC_USER_NAME}:${SC_USER_NAME} /entrypoint.sh && \
-  chown -R ${SC_USER_NAME}:${SC_USER_NAME} /usr/local/bin/static-web-server && \
-  chown -R ${SC_USER_NAME}:${SC_USER_NAME} /public
+RUN chown -R "${SC_USER_NAME}:${SC_USER_NAME}" /entrypoint.sh && \
+  chown -R "${SC_USER_NAME}:${SC_USER_NAME}" /usr/local/bin/static-web-server && \
+  chown -R "${SC_USER_NAME}:${SC_USER_NAME}" /public
+
 
 USER ${SC_USER_NAME}
 
 FROM base as build
+# front-end client last (image name is the path to the Dockerfile)
+COPY --from=client/tools/qooxdoo-kit/builder:latest --chown=${SC_USER_NAME}:${SC_USER_NAME} \
+  /project/build-output "/static-content"
 ENV SC_BUILD_TARGET build
 
-FROM base as production
+FROM build as production
 ENV SC_BUILD_TARGET production
 
 FROM base as development
```

Comment:
so this avoids ``/project/build-output `` in development (which is the right thing since we are mounting the client source volume) .. How long did we have this? This will definively make a speed difference

Reply:
BTW< why even have a build stage??? What does it bring?

Reply:
how long? well I guess this is here for a very long time.

Reply:
> how long? well I guess this is here for a very long time.

since it was added, since I only made it work and this corner case slipped me